clang-tools-extra/test/clang-doc/enum.cpp

```diff
@@ -0,0 +1,38 @@
+// RUN: clang-doc --format=html --doxygen --output=%t/docs --executor=standalone %s
+// RUN: clang-doc --format=md --doxygen --output=%t/docs --executor=standalone %s
```

Comment:
This can't work right, can it? you haven't made `%t`, you also don't need `docs`, right? It would be good if I didn't have to point these out in every patch... I've given this piece of feedback in almost all of your patches.

Reply:
sorry I've realize originally that we didn't need to explicitly mkdir but its probably better to explicitly specify it